content/networking/security-groups.md

```diff
@@ -1,70 +1,43 @@
 /*
 Title: Security Groups
+Sort: 0
 */
 
-By default, a raw instance only has port 22 open so you can SSH in after it's
-created. We don't want to expose your servers to anything a brand new instance
-isn't prepared for. To manage access, we use Security Groups. The best practise 
-for security groups, is to make them as specific as possible. A single instance
-can belong to multiple security groups, so for a web server, you might want
-ports 22, 443, and 80 open. Or for a DB server you might want port 5432 open
-for PostgreSQL.
+Network Security Groups allow you to open traffic on a port-by-port basis. 
+This flexible model can extend to create any number of situations, whether
+simple or complex while maintaining strict security.
 
-If access is attempted to a port/IP combination that is not assigned, no
-connection will be established at all and the connection will time out.
+## Restricted by Default
 
-The default security group has just port 22 open, so we'll create a brand new
-one named "Web" for a web server as an example.
+By default, a raw instance only has port 22 open so you can SSH in after 
+it's created. We don't want to expose your servers to anything a brand new 
```

Comment:
Maybe elaborate a little bit on the security implications a little further. "Exposing all ports, or ports which don't need to be exposed, can lead to unauthorized access, etc."